packages/cosmic-proto/package.json

```diff
@@ -133,7 +133,9 @@
     "codegen": "./update-protos.sh && node scripts/codegen.cjs",
     "prepare": "npm run build",
     "lint-fix": "yarn lint:eslint --fix",
-    "lint": "tsc",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc",
+    "lint:eslint": "eslint $(sed 's/#.*//; /[^[:space:]]/ s/^/--ignore-pattern /' .eslintignore) .",
```

Comment:
This is unnecessarily hacky. The higher level config is at project root, so if that's causing problems we should fix it there.

But I'm not sure any of this is worth the effort. What do you want to eslint in this package? It's mostly codegen.

Reply:
> What do you want to eslint in this package? It's mostly codegen.

I encountered an issue here while working on another PR, but I don't remember where.

> This is unnecessarily hacky. The higher level config is at project root, so if that's causing problems we should fix it there.

OK, fair point. Done.